src/mt/process-list.cpp

```diff
@@ -2,6 +2,7 @@
  * Author: O. Stasse, 2019
  * See LICENSE file in the root directory of this repository.
  */
+#include <iostream>
```

Comment:
If possible, dynamic-graph should not use `iostream`. The reason is that it can introduce delay in the control loop of the robot. If you need to some debugging output, the best option is to use the real-time logging mechanism.

I don't think the usage of `cout` in this file is necessary. Could you remove this include and the line where `cout` is used ?

Reply:
Yup I can, It's not a part of my code so I just fix the problem by using iostream